lms/templates/navigation.html

```diff
@@ -85,7 +85,8 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
               </a>
             </li>
             <li class="tab-nav-item">
-              <a class="${'active ' if reverse('program_listing_view') == request.path else ''}tab-nav-link" href="${reverse('program_listing_view')}">
+              ## TODO: Update to user actual program ids
+              <a class="${'active ' if (reverse('program_listing_view') == request.path) or (reverse('program_details_view', args=[1]) == request.path) else ''}tab-nav-link" href="${reverse('program_listing_view')}">
```

Comment:
Alternatively, you could pluck the program identifier out of the path and use that in a call to reverse. I don't like this option as much because it's a lot of Python to include in a template.